evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -1759,6 +1759,61 @@ def test_explain_predictions_oversampler(estimator, algorithm, fraud_100):
     assert report["feature_values"].isnull().sum() == 0
 
 
+@pytest.mark.noncore_dependency
+@pytest.mark.parametrize("problem_type", [ProblemTypes.MULTICLASS, ProblemTypes.BINARY])
+def test_explain_predictions_class_name_matches_class_name_in_y(
+    problem_type, fraud_100
+):
+    if problem_type == ProblemTypes.BINARY:
+        X, y = fraud_100
```

Comment:
omega-nit-pick but `X, y = fraud_100` is a duplicate line for both conditions and can be moved out? :)